File: python/LeetCode_344_ReverseString.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        low = 0
        high = len(s) - 1

        while (low < high):
            s[low], s[high] = s[high], s[low]
            low += 1
            high -= 1

        

File: c/LeetCode_344_ReverseString.c
void reverseString(char* s, int sSize) {
    if (sSize <= 0) return;

    int low = 0;
    int high = sSize - 1; 

    while (low < high) {
        char tmp = s[low];
        s[low] = s[high];
        s[high] = tmp;

        ++low;
        --high;
    }
}